docs/run-sparkbench.md

```diff
@@ -88,3 +88,59 @@ hibench.yarn.executor.num   |   Spark executor number in Yarn mode
 spark.executor.memory  | Spark executor memory
 spark.driver.memory    | Spark driver memory
+
+
+### 8. Run xgboost workload ###
+
+Hibench xgboost benchmark depends on the xgboost libraries to build and run. The libs are ```xgboost4j_<scala version>-<xgboost version>.jar``` and ```xgboost4j-spark_<scala version>-<xgboost version>.jar```.<br>
+The relevant configurations are in ```./sparkbench/ml/pom.xml```
+```
+    <dependency>
+      <groupId>ml.dmlc</groupId>
+      <artifactId>xgboost4j_${scala.binary.version}</artifactId>
+      <version>1.1.0</version>
+    </dependency>
+    <dependency>
+      <groupId>ml.dmlc</groupId>
+      <artifactId>xgboost4j-spark_${scala.binary.version}</artifactId>
+      <version>1.1.0</version>
+    </dependency>
+```
+and ```./pom.xml```
+```
+    <repository>
+      <id>xgboostrepo</id>
+      <name>XGBoost Maven Repo</name>
+      <url>https://s3-us-west-2.amazonaws.com/xgboost-maven-repo/release</url>
+      <releases>
+        <enabled>true</enabled>
+      </releases>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+    </repository>
+```
+
+#### 8.a latest xgboost release (default) ####
+
+By default, the hibench xgboost benchmark is configured to use the latest xgboost release from https://s3-us-west-2.amazonaws.com/xgboost-maven-repo/release.<br>
+To use it, simply build hibench, prepare data and run xgboost benchmark. For example,
+```
+$ mvn -Psparkbench -Dmodules -Pml -Dspark=2.4 -Dscala=2.12 clean package
+$ bin/workloads/ml/xgboost/prepare/prepare.sh && hdfs dfs -du -s -h /HiBench/XGBoost/Input
+$ bin/workloads/ml/xgboost/spark/run.sh
+```
+
+#### 8.b other xgboost releases ####
+
+To use other xgboost releases, change the xgboost versions for xgboost4j and xgboost4j-spark to the target versions in ```./sparkbench/ml/pom.xml```. The ```scala.binary.version``` can be specified by command line parameter ```-Dscala```.<br>
+e.g. to use xgboost v1.0.0, change ```<version>1.1.0</version>``` to ```<version>1.0.0</version>``` for both xgboost4j and xgboost4j-spark.<br>
+If the xgboost release is from other maven repo, update the xgboostrepo url in ```./pom.xml``` as well.<br>
+After that, build hibench, prepare data and run xgboost benchmark.
+
+#### 8.c xgboost jar files ####
+
+If you only have the xgboost jar files, just copy them to $SPARK_HOME/jars/ and update the relevant versions for xgboost4j and xgboost4j-spark in sparkbench/ml/pom.xml to get aligned.<br>
+For example, if xgboost is built from source on a Linux platform, the jars will be generated and installed to ```~/.m2/repository/ml/dmlc/xgboost4j_<scala version>/<xgboost version>-SNAPSHOT/``` and ```~/.m2/repository/ml/dmlc/xgboost4j-spark_<scala version>/<xgboost version>-SNAPSHOT/``` respectively. To use them, copy the 2 jars to $SPARK_HOME/jars/ and update the relevant versions for xgboost4j and xgboost4j-spark in the pom.xml files.<br>
+After that, build hibench, prepare data and run xgboost benchmark.
+
```

Comment:
Generally, the doc style is not consistent as the original doc. and too complicated to follow. 
I suggest rewrite or remove. We can merge code first. It should be runnable with default setting.